case-1/src/main/java/com/company/appconfig/Main.java

```diff
@@ -0,0 +1,13 @@
+package com.company.appconfig;
+
+import com.company.app.PvModuleLayouts;
+import com.company.app.pvmodulelayouts.DefaultPvModuleLayouts;
+import com.company.app.pvmodulelayouts.layoutspecification.RoofSlopeDependentLayoutSpecification;
+
+// @Configuration
+public class Main {
+    // @Bean
+    public PvModuleLayouts pvModuleLayouts() {
+        return new DefaultPvModuleLayouts(RoofSlopeDependentLayoutSpecification::new);
+    }
+}
```

Comment:
This code that would plug everything together goes into a parallel package hierarchy